docs/conf/portland/2021/news/announcing-schedule.rst

```diff
@@ -0,0 +1,61 @@
+:template: {{year}}/generic.html
+
+.. post:: Mar 17, 2020
+   :tags: {{year}}, portland-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're just over a month away from the online Portland conference, and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familar with.
+
+Today we're announcing the full presentation schedule and the platform we'll be hosting the conference on.
+We are working on last-minute details for a few more activities during the conference, so look out for more information in our next mailing.
+
+Conference platform
+--------------------
+
+Following the success of last year's virtual format, the conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. We're updating our Welcome Wagon guide, to be released soon, so it'll all feel familiar before you've even arrived.
```

Comment:
Should we announce the social event platform here? https://spatial.chat/

Reply:
I feel like you're the only person who knows much about it. Wanna add a para? @ericholscher